src/service/xhr-impl.js

```diff
@@ -223,11 +223,11 @@ function setupJson_(init) {
   init.headers = init.headers || {};
   init.headers['Accept'] = 'application/json';
 
-  if (init.method == 'POST') {
+  if (init.method == 'POST' && !isFormData(init.body)) {
```

Comment:
The below is basically for json-body requests, we'd like to send `FormData` instead of JSON body to enable normal submission backend to handle it without having to handle JSON.